src/platform-implementation-js/dom-driver/gmail/views/gmail-route-view/gmail-route-view.ts

```diff
@@ -99,6 +100,8 @@ class GmailRouteView {
   }
 
   destroy() {
+    this.#destroyed = true;
```

Comment:
If we add a `#destroyed` field, does the check [here](https://github.com/InboxSDK/InboxSDK/pull/1086/files#diff-bfcc9a881cfa869791111b1e104c99a8318a2273b4420ec1a74d20b25d4291d4R302) need to change?

I wonder if the `asap` call wrapping an async callback is load bearing, but I doubt it.

Reply:
adding a check for `#destroyed` is not totally required in this case. Will keep this part unchanged to minimize the diff.